numpy/_core/numeric.pyi

```diff
@@ -428,14 +428,7 @@ def full_like(
 @overload
 def count_nonzero(
     a: ArrayLike,
-    axis: None = ...,
-    *,
-    keepdims: L[False] = ...,
-) -> int: ...
-@overload
-def count_nonzero(
-    a: ArrayLike,
-    axis: _ShapeLike = ...,
+    axis: None | _ShapeLike = ...,
```

Comment:
why not keep the overloads and return `np.intp` instead of `int` from the first one?

Reply:
True, that's better, done!